### PR TITLE
Add HautelookAliceBundle mention

### PR DESCRIPTION
**Pull request by theofidry**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | 2.3+ |

HautelookAliceBundle is a bundle for Faker and Alice so the integration might be easier than using the two libraries.

### CODE REVIEW

**Comment by xabbuh** [comment]
Thank you for proposing this change @theofidry. However, there are also bundles to integrate faker (like https://github.com/willdurand/BazingaFakerBundle). Should we mention them here too?

**Comment by theofidry** [comment]
Obviously we can't mention them all:

* [HautelookAliceBundle](https://github.com/hautelook/AliceBundle)
* [h4cc/AliceFixturesBundle](https://github.com/h4cc/AliceFixturesBundle)
* [KnpLabs/rad-fixtures-load](https://github.com/KnpLabs/rad-fixtures-load)
* [LafourchetteFixturesBundle](https://github.com/lafourchette/FixturesBundle)
* [DoctrineFixturesBundle](https://github.com/doctrine/DoctrineFixturesBundle)

Plus the bundles for Faker... However the three most popular in the order is DoctrineFixturesBundle (no longer active), HautelookAliceBundle and BazingaFakerBundle.

But that's just a proposal, if you feel it wouldn't be fair to other bundles well at least I would have tried :)

**Comment by dupuchba** [comment]
:+1: for mentioning them all but don't forget that when there is too much choices, people get lost :-). Maybe it would be useful to suggest one/two well proofed library.

**Comment by theofidry** [comment]
The issue there is that there is yet to be a consensus on the choice of the bundle. To me they all look redundant and I've been a long time frustrated by how things were not moving before rewriting HautelookAliceBundle.

**Comment by stof** [comment]
DoctrineFixturesBundle does not integrate Alice nor Faker (you can use them inside your doctrine fixtures though)

**Comment by theofidry** [comment]
@stof true, but it was more about having advanced fixtures rather than actually using Alice and Faker.

**Comment by javiereguiluz** [comment]
I like the idea of mentioning only one bundle to integrate Alice/Faker. We should choose the best one according to our criteria. If we can't reach an agreement, let's mention two bundles.

I'd even reword the original message. Instead this:

```
Consider using `Faker`_, `Alice`_ or `HautelookAliceBundle`_ libraries to generate
real-looking data [...]
```

We could say something like:

```
Consider using `HautelookAliceBundle`_ to generate real-looking data for your fixtures.
This bundle integrates the `Faker`_ and `Alice`_ libraries [...]
```

File: best_practices/tests.rst [hunk 1]
```diff
@@ -113,7 +113,7 @@ pure JavaScript-based testing tools.
 Learn More about Functional Tests
 ---------------------------------
 
-Consider using `Faker`_ and `Alice`_ libraries to generate real-looking data
+Consider using `Faker`_, `Alice`_ or `HautelookAliceBundle`_ libraries to generate real-looking data
```
Comment (timglabisch) [inline best_practices/tests.rst]: it's not a library, its a bundle.

**Comment by WouterJ** [comment]
Hi @theofidry, thanks for starting this pull request. I've taken your commit and applied @javiereguiluz's comment and put both commits in a new PR: https://github.com/symfony/symfony-docs/pull/6051 Although I close this PR, you'll still get the credits for this suggestion.

Have a nice weekend! :christmas_tree:

**Pull request closed without merging**